deps/v8/src/base/platform/platform-posix.cc

```diff
@@ -370,12 +387,16 @@ bool OS::Release(void* address, size_t size) {
   DCHECK_EQ(0, size % CommitPageSize());
   return munmap(address, size) == 0;
 }
+#endif // !V8_OS_ZOS
 
 // static
 bool OS::SetPermissions(void* address, size_t size, MemoryPermission access) {
   DCHECK_EQ(0, reinterpret_cast<uintptr_t>(address) % CommitPageSize());
   DCHECK_EQ(0, size % CommitPageSize());
 
+#if defined(V8_OS_ZOS)
+  return true;	
+#else
```

Comment:
We may need to revisit this or add a `//TODO` comment since mprotect is available on z/OS: https://www.ibm.com/support/knowledgecenter/SSLTBW_2.2.0/com.ibm.zos.v2r2.bpxb100/mpr.htm